repos/c2sm/packages/flexpart-cosmo/package.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+#
+from spack import *
+from distutils.dir_util import copy_tree
+
+
+class FlexpartCosmo(MakefilePackage):
+    """flexpart is a Lagrangian dispersion model"""
+
+    homepage = 'https://github.com/C2SM-RCM/flexpart'
+    url = 'https://github.com/C2SM-RCM/flexpart/archive/refs/tags/V8C3-preop.tar.gz'
+    git = 'ssh://git@github.com/C2SM-RCM/flexpart.git'
+
+    version('master', branch='master')
+
+    depends_on('eccodes@2.19.0 jp2k=none +fortran')
+    depends_on('netcdf-fortran')
+    depends_on('jasper@1.900.1')
```

Comment:
According to @pirmink, Jasper isn't really a dependency of flexpart-cosmo. Maybe double check (with him).